### ISSUE TITLE
WCAG用語: CSS ピクセル (CSS pixel) の見直し

### ISSUE THREAD

**Issue by jidaikobo-shibata** [waic/wcag22#1448]
ID: `dfn-css-pixels`
  この用語について検討します。
  [CSS ピクセル (CSS pixel)](https://waic.jp/translations/WCAG22/#dfn-css-pixels)

**Comment by momdo** [comment]
> CSS ピクセルは、CSS におけるすべての長さ及び測定のための規範的な測定単位である。この単位は密度非依存で、ディスプレイに存在する実際のハードウェアピクセルとは異なる。ユーザエージェント及びオペレーティングシステムは CSS ピクセルが、ディスプレイの物理的な寸法と予想される視距離 (コンテンツ制作者は決定できない要因) を考慮した CSS Values and Units Module Level 3 reference pixel [css3-values] にできるだけ沿うように設定されていることを確認すべきである。

> A CSS pixel is the canonical unit of measure for all lengths and measurements in CSS. This unit is density-independent, and distinct from actual hardware pixels present in a display. User agents and operating systems should ensure that a CSS pixel is set as closely as possible to the CSS Values and Units Module Level 3 reference pixel [css3-values], which takes into account the physical dimensions of the display and the assumed viewing distance (factors that cannot be determined by content authors).

measurementsは「測定」でよいか？「寸法」？

CSS ピクセルは、CSS における全ての長さ及び寸法のための規範的な測定単位である。

---

ensure「保証する」
closelyは意訳になるが「近似して」とするのはどうか。

> CSS Values and Units Module Level 3 reference pixel [css3-values]

これは「CSS Values and Units Module Level 3 のreference pixel」ということになるが、このままでよいか？
例えば「CSS Values and Units Module Level 3 の参照ピクセル (reference pixel)」と翻訳するのが適切ではなかろうか？あと、非制限用法の関係代名詞なので、分けて訳すのがよいか。こうか？

ユーザエージェント及びオペレーティングシステムは、CSS ピクセルが、CSS Values and Units Module Level 3 の参照ピクセル (reference pixel) [css3-values] に可能な限り近似して設定されていることを保証すべきである。
この参照ピクセルは、ディスプレイの物理的な寸法と想定される視距離 (コンテンツ制作者が決定できない要因) を考慮している。
